tests/phpunit/tests/script-modules/wpScriptModules.php

```diff
@@ -512,47 +512,55 @@ public function test_wp_enqueued_script_modules_with_dependants_add_import_map()
 	}
 
 	/**
-	 * Tests the functionality of the `get_versioned_src` method to ensure
+	 * Tests the functionality of the `get_src` method to ensure
 	 * proper URLs with version strings are returned.
 	 *
 	 * @ticket 56313
 	 *
-	 * @covers ::get_versioned_src()
+	 * @covers ::get_src()
 	 */
-	public function test_get_versioned_src() {
-		$get_versioned_src = new ReflectionMethod( $this->script_modules, 'get_versioned_src' );
-		$get_versioned_src->setAccessible( true );
+	public function test_get_src() {
```

Comment:
What about adding an `add_filter()` call to test that the filter applies as expected and passes the expectant args?

Reply:
I didn't see much point personally in testing that a filter works; but happy to add one.

Reply:
Added.

Reply:
Since it is the API surface for plugins to use, I see it as a way to make sure that code changes in core don't break plugins that expect certain args to be passed to a filter.